services/api-server/src/simcore_service_api_server/models/schemas/jobs.py

```diff
@@ -130,7 +134,7 @@ def compute_results_checksum(self):
 
 
 class Job(BaseModel):
-    id: UUID
+    id: JobID  # noqa: A003
```

Comment:
also here, what is this and is there not a human readable error?

Reply:
same as above. See new `ruff` tooling